framework/src/actions/CommonOutputAction.C

```diff
@@ -138,6 +138,8 @@ CommonOutputAction::validParams()
                         "Enable/disable printing of the linear solver convergence reason to the "
                         "screen. This parameter only affects the output of the third-party solver "
                         "(e.g. PETSc), not MOOSE itself.");
+  params.addParam<bool>("output_screen", true, "Output to the screen for console outputs");
+  params.addParam<bool>("output_file", false, "Output to the file for console outputs");
```

Comment:
are these two params used?

Reply:
Yes, it will be picked up when this action is adding `Console`. I am pretty sure, but cannot find it quickly, must be in that `create` function somewhere.

Reply:
I tried `output_screen=false` in a multiapp input, and it worked.

Reply:
This name collides with the parameter names in `DOFMapOutput`, so I had to change this commit a little bit.